### PR TITLE
 Resolve issue by adding changelog.js file in user

### LINKED ISSUE #5716: [FEATURE] User changelog entry for Profile progress
### Description

Now we have a profile progress, we should let the users now about it in the user changelog page
https://github.com/EddieHubCommunity/LinkFree/blob/main/pages/changelog.js

Screenshot 2023-03-30 at 13 04 04

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by ApurvaBasule**
Closes #5716  

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5727"></a>

### CODE REVIEW

**Comment by ApurvaBasule** [comment]
closes #5727

**Comment by ApurvaBasule** [comment]
#5727 closes

File: .env.example [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
+NEXT_PUBLIC_GA_MEASUREMENT_ID="-"
 NEXT_PUBLIC_BASE_URL="http://localhost:3000"
 NODE_ENV="development"
 
-GITHUB_ID=""
-GITHUB_SECRET=""
-NEXTAUTH_SECRET=""
+GITHUB_ID="10c01b64991cf71735d1"
+GITHUB_SECRET="c2a58bbcea3cef1dd3c847eea392e0f2369b39f2"
+NEXTAUTH_SECRET="afsfdsafadsf"
```
Comment (eddiejaoude) [inline .env.example:8]: You should not commit your secrets here, please remove these changes and reset your GitHub OAuth secrets

File: pages/changelog.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
+import { CalendarIcon } from "@heroicons/react/20/solid";
```
Comment (eddiejaoude) [inline pages/changelog.js:1]: The file already exists, you only need to add a new object entry here 
https://github.com/EddieHubCommunity/LinkFree/blob/main/pages/changelog.js#L12

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Thank you for your contribution, I have added some inline comments

Also please do not commit the `package-lock.json` file if you have not made changes to the `package.json` file

**Review by bharat5604 — CHANGES_REQUESTED** [review]
Remove secret tokens

**Comment by eddiejaoude** [comment]
> Remove secret tokens

@bharat5604 thank you for reviewing, please read existing comments first as this has already been mentioned here https://github.com/EddieHubCommunity/LinkFree/pull/5727#pullrequestreview-1366114316 , repeating the same comments does not help, can scare off contributors and also creates more notifications for everyone

**Pull request closed without merging**